randomGenerator.cpp

```diff
@@ -0,0 +1,16 @@
+/*
+ * randomGenerator.cpp
+ *
+ *  Created on: Mar 18, 2019
+ *      Author: dungnguyen
+ */
+#include <time.h>
+#include <cstdlib>
+
+int randomGenerator(int toNum)	{
+	srand(time(NULL));
```

Comment:
"srand" needs to be called outside of the function, maybe at the beginning of main. And it should only be called once. Right now, every time you call the randomGenerator function, the same number is generated.